package.json

```diff
@@ -21,7 +21,8 @@
     "serve": "gatsby serve",
     "preview": "npm run build && npm run serve",
     "pretty": "pretty-quick --pattern '**/*.{js,mdx}'",
-    "deploy:beta": "npm run build && surge public --domain https://h2p-beta.surge.sh"
+    "deploy:beta:surge": "surge public --domain https://h2p-beta.surge.sh",
+    "deploy:beta": "npm run build && npm run deploy:beta:surge"
```

Comment:
Could take the `npm run build &&` out of this and run build for both.

Reply:
In fact, could probably just remove this.

Reply:
The initial thoughts when doing this was to have everything keep behaving the same as before this commit, so `deploy:beta` still does build+deploy, but the main workflow will only actually build once even if it ends up pushing to 2 different places (docker + surge).

Given that backwards-compat isn't really needed, I'd probably change to just have `deploy:surge` or so and scrap `deploy:beta` so that you had to run `npm build && npm deploy:surge` or so to actually push it.